compose/auth_proxy.yml

```diff
@@ -5,12 +5,14 @@ services:
       - "${CRYOSTAT_HTTP_PORT}"
     environment:
       CRYOSTAT_HTTP_PROXY_HOST: auth
-      CRYOSTAT_HTTP_PROXY_PORT: '8080'
-      QUARKUS_HTTP_PROXY_PROXY_ADDRESS_FORWARDING: 'true'
-      QUARKUS_HTTP_PROXY_ALLOW_X_FORWARDED: 'true'
-      QUARKUS_HTTP_PROXY_ENABLE_FORWARDED_HOST: 'true'
-      QUARKUS_HTTP_PROXY_ENABLE_FORWARDED_PREFIX: 'true'
-      QUARKUS_HTTP_PROXY_TRUSTED_PROXIES: 127.0.0.1:${CRYOSTAT_HTTP_PORT}
+      CRYOSTAT_HTTP_PROXY_PORT: "${CRYOSTAT_PROXY_PORT}"
+      CRYOSTAT_HTTP_PROXY_TLS_ENABLED: "true"
```

Comment:
This should be somehow dependent on `CRYOSTAT_PROXY_PROTOCOL` - maybe it should also be set/overridden in `auth_proxy_https.yml`?